presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/AggregationMatcher.java

```diff
@@ -67,7 +67,15 @@ public MatchResult detailMatches(PlanNode node, StatsProvider stats, Session ses
             return NO_MATCH;
         }
 
-        if (groupingSets.size() != aggregationNode.getGroupingSets().size()) {
+        if (!matches(groupingSets.getGroupingKeys(), aggregationNode.getGroupingKeys(), symbolAliases)) {
```

Comment:
These conditions feel pretty close to an equality check on GroupingSetsDescriptor

Reply:
Yeah, but it needs access to symbolAliases.